nx/blocks/importer/index.js

```diff
@@ -24,7 +26,13 @@ async function findFragments(pageUrl, text, liveDomain) {
 
   const dom = parser.parseFromString(text, 'text/html');
   const results = dom.body.querySelectorAll(LINK_SELECTORS.join(', '));
-  const linkedImports = [...results].reduce((acc, a) => {
+  const matches = text.match(LINK_SELECTOR_REGEX)?.map((svgUrl) => {
```

Comment:
Thought about combining this and having a bigger regex, but with relative links, fragments, etc. this would turn out to be quite ugly and unreadable